Web/Edubase.Web.UI/403.aspx

```diff
@@ -212,11 +213,13 @@
                 <div class="footer-meta">
                     <div class="footer-meta-inner">
                         <h2 class="visuallyhidden">Support links</h2>
-                        <ul>
-                            <li><a href="/cookies">Cookies</a></li>
-                            <!--<li><a href="#">Contact</a></li>-->
-                            <li><a href="/guidance">Guidance</a></li>
-                            <li><a href="/news">News</a></li>
+                        <ul class="govuk-footer__inline-list">
+                            <li class="govuk-footer__inline-list-item"><a href="https://form.education.gov.uk/service/Data-collections-service-request-form" target="_blank" rel="noreferrer noopener" class="govuk-footer__link">Contact (opens in new tab)</a></li>
```

Comment:
Did you mean to update the Contact link as well, as dont think this was expected as part of this story (and the general page footer directs users to the contact page not needing to open in a new window)